src/layouting/data_limits.jl

```diff
@@ -108,7 +108,18 @@ function text_limits(x::AbstractVector)
 end
 
 function atomic_limits(x::Text{<: Tuple{Arg1}}) where Arg1
-    return boundingbox(x)
+    if x.space[] == :data
+        return boundingbox(x)
+    elseif x.space[] == :screen
+        if x.position[] isa Union{StaticArrays.StaticArray, Tuple{Real, Real}, GeometryBasics.Point}
```

Comment:
> `Tuple{Real, Real}`

should we make this `NTuple{2, Real}, NTuple{3, Real}`?

Reply:
no it's actually annoying if things don't work if you give a `position = (1, 2.3)`

Reply:
Ah, makes sense.  We should probably generalize to the 3D case as well (`Tuple{Real, Real, Real}`).